qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -3,32 +3,37 @@
     start_vm = yes
     kill_vm_on_error = yes
     check_guest_bsod = yes
-    login_timeout = 240
+    login_timeout = 360
     suppress_exception = no
     run_bgstress = rng_bat
-    session_cmd_timeout = 240
-    read_rng_cmd = "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
-    rng_data_rex = "0x\w"
-    driver_name = "viorng"
+    session_cmd_timeout = 360
     cdrom_cd1 = isos/windows/winutils.iso
     no no_virtio_rng
-    only Windows
+    Windows:
+        driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
+        target_process = random\w*.exe
+        rng_data_rex = "0x\w"
+        driver_name = "viorng"
+        read_rng_cmd = for /l %i in (1, 1, 1000) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
```

Comment:
And also check windows running status in AMD host,jobs can pass:

# lscpu 
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                64
On-line CPU(s) list:   0-63
Thread(s) per core:    2
Core(s) per socket:    8
Socket(s)：                 4
NUMA node(s):       8
Vendor ID：           AuthenticAMD
CPU family：          21
Model：              2
Moder name：        AMD Opteron(tm) Processor 6380
Stepping：              0
CPU MHz：             2493.937
BogoMIPS：            4987.87
Virtualization：           AMD-V
L1d cache：          16K
L1i cache：          64K
L2 cache：           2048K
L3 cache：           6144K
NUMA node0 CPU：    0,4,8,12,16,20,24,28
NUMA node1 CPU：    32,36,40,44,48,52,56,60
NUMA node2 CPU：    1,5,9,13,17,21,25,29
NUMA node3 CPU：    33,37,41,45,49,53,57,61
NUMA node4 CPU：    2,6,10,14,18,22,26,30
NUMA node5 CPU：    34,38,42,46,50,54,58,62
NUMA node6 CPU：    35,39,43,47,51,55,59,63
NUMA node7 CPU：    3,7,11,15,19,23,27,31
Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc art rep_good nopl nonstop_tsc extd_apicid amd_dcm aperfmperf pni pclmulqdq monitor ssse3 fma cx16 sse4_1 sse4_2 popcnt aes xsave avx f16c lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs xop skinit wdt lwp fma4 tce nodeid_msr tbm topoext perfctr_core perfctr_nb cpb hw_pstate bmi1 arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfthreshold

'01-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_stop_continue.before_bg_test' Finshed 2017-08-09 18:07:40 2017-08-09 18:16:31 PASS 530.592321
'02-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_stop_continue.during_bg_test' Finshed 2017-08-09 18:16:31 2017-08-09 18:20:26 PASS 235.514572
'03-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_stop_continue.after_bg_test' Finshed 2017-08-09 18:20:26 2017-08-09 18:24:16 PASS 229.674035
'04-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_shutdown.before_bg_test' Finshed 2017-08-09 18:24:16 2017-08-09 18:29:36 PASS 319.890531
'05-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_shutdown.during_bg_test' Finshed 2017-08-09 18:29:36 2017-08-09 18:31:32 PASS 115.63577
'06-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_shutdown.after_bg_test' Finshed 2017-08-09 18:31:32 2017-08-09 18:35:03 PASS 210.762868
'07-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_reboot.before_bg_test' Finshed 2017-08-09 18:35:03 2017-08-09 18:40:01 PASS 298.463096
'08-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_reboot.during_bg_test' Finshed 2017-08-09 18:40:01 2017-08-09 18:43:56 PASS 234.720888
'09-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_reboot.after_bg_test' Finshed 2017-08-09 18:43:56 2017-08-09 18:49:30 PASS 333.855864
'10-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_system_reset.before_bg_test' Finshed 2017-08-09 18:49:30 2017-08-09 18:54:51 PASS 321.300461
'11-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_system_reset.during_bg_test' Finshed 2017-08-09 18:54:51 2017-08-09 19:02:52 PASS 481.299938
'12-smp_32.262144m.repeat1.run_test.Host_RHEL.m7.u4.virtio_rng.rng_random.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.viorng_in_use.with_system_reset.after_bg_test' Finshed 2017-08-09 19:02:52 2017-08-09 19:08:21 PASS 328.78651

@suqinhuang